draft-ietf-masque-h3-datagram.md

```diff
@@ -199,12 +199,11 @@ CAPSULE allows reliably sending request-related information end-to-end, even in
 the presence of HTTP intermediaries.
 
 CAPSULE is an HTTP/3 Frame (as opposed to a QUIC frame) which SHALL only be
-sent in client-initiated bidirectional streams. Intermediaries MUST forward all
-received CAPSULE frames in their unmodified entirety on the same stream where
-it would forward DATA frames. Intermediaries MUST NOT send any CAPSULE frames
-other than the ones it is forwarding. Intermediaries respect the order of
-CAPSULE frames: if an intermediary receives two CAPSULE frames in a given
-order, it MUST forward them in the same order.
+sent in client-initiated bidirectional streams. Intermediaries forward received
```

Comment:
Not specific to this PR but the changes made me realise we probably want to specific what receivers of CAPSULE on the wrong type of stream do. I.E. if I understand CAPSULE and I see one arrive on the peer's control stream, I probably should be closing the connection with H3_FRAME_UNEXPECTED. Maybe we should spin this off to a new issue?

Reply:
Fixed